sbi.adoc

```diff
@@ -1,3 +1,32 @@
 [[sbi]]
 == SBI Requirements
 
+The supervisor binary interface (SBI) cite: [SBI] specification defines the interface
+between the supervisor mode and the next higher privilege mode. This section
+defines the mandatory SBI version and extensions implemented by the higher
+privilege mode in order to be compatible with this specification.
+
+=== Minimum SBI version
+The SBI implementation must be v1.0 or higher. 
+
+=== Mandatory SBI extensions
+
+The SBI v1.0 allows optional SBI extensions. However, the following optional
+extensions are mandatory for a compliant system. 
+
+* SBI Hart State Management (HSM)
+
+* SBI System Reset (SRST)
```

Comment:
Is that truly mandatory? OSes are going to be mandated (by this spec) to always use UEFI RT call to perform a reset or shudown, and conceptually it doesn't matter if the UEFI RT SystemReset call is backed by SRST or something else.

Reply:
Having SRST avoids having to implement system reset drivers in the UEFI firmware. In the ARM world PSCI is also on the TF-A level.

Reply:
We can add more clarification saying UEFI firmware (running in S-mode) must implement the ResetSystem service via SBI SRST extension.

Reply:
I have updated the text. Please let me know if the clarification is sufficient.

Reply:
The below clarification is meant for both suspend and reset.

Reply:
I think I was confused, and it looked like a new PR was created. I think we're all good. Thanks, Atish.